.git-blame-ignore-revs

```diff
@@ -0,0 +1,2 @@
+# change column limit to 80
+39cd0478af477b63ecc925fa8af8b0c82e4566e6
```

Comment:
Is this file needed? What does this do?

Reply:
git blame ignores changes made by commits mentioned in this file. Mainly used to prevent git blame from getting trashed during mass-formatting commits.